evalml/preprocessing/utils.py

```diff
@@ -55,16 +63,15 @@ def split_data(X, y, regression=False, test_size=.2, random_state=None):
     X = _convert_to_woodwork_structure(X)
     y = _convert_to_woodwork_structure(y)
 
-    if regression:
-        CV_method = ShuffleSplit(n_splits=1,
-                                 test_size=test_size,
-                                 random_state=random_state)
-    else:
-        CV_method = StratifiedShuffleSplit(
-            n_splits=1,
-            test_size=test_size,
-            random_state=random_state)
-    train, test = next(CV_method.split(X.to_dataframe(), y.to_series()))
+    data_splitter = None
+    if is_time_series(problem_type):
+        data_splitter = TrainingValidationSplit(test_size=test_size, shuffle=False, stratify=None, random_state=random_state)
```

Comment:
This is the change @freddyaboulton and I were discussion. Once this PR is merged we can update looking glass to use `split_data`